src/SFML/Graphics/Font.cpp

```diff
@@ -375,30 +376,37 @@ bool Font::hasGlyph(Uint32 codePoint) const
 
 
 ////////////////////////////////////////////////////////////
-float Font::getKerning(Uint32 first, Uint32 second, unsigned int characterSize) const
+float Font::getKerning(Uint32 first, Uint32 second, unsigned int characterSize, bool bold) const
 {
     // Special case where first or second is 0 (null character)
     if (first == 0 || second == 0)
         return 0.f;
 
     FT_Face face = static_cast<FT_Face>(m_face);
 
-    if (face && FT_HAS_KERNING(face) && setCurrentSize(characterSize))
+    if (face && setCurrentSize(characterSize))
     {
         // Convert the characters to indices
         FT_UInt index1 = FT_Get_Char_Index(face, first);
         FT_UInt index2 = FT_Get_Char_Index(face, second);
 
+        // Retrieve position compensation deltas generated by FT_LOAD_FORCE_AUTOHINT flag
+        float firstRsbDelta = getGlyph(first, characterSize, bold).rsb_delta;
+        float secondLsbDelta = getGlyph(second, characterSize, bold).lsb_delta;
+
         // Get the kerning vector
         FT_Vector kerning;
-        FT_Get_Kerning(face, index1, index2, FT_KERNING_DEFAULT, &kerning);
+        kerning.x = kerning.y = 0;
+        if( FT_HAS_KERNING(face))
```

Comment:
Should be `if (`

Reply:
@eXpl0it3r Thanks for spotting that. I'm usually pedantic when it comes to formatting conventions.